src/cdk-lib/bedrock/agent.ts

```diff
@@ -380,13 +380,13 @@ export class Agent extends Construct {
    * A list of action groups associated with the agent
    * @private
    */
-  public actionGroups: bedrock.CfnAgent.AgentActionGroupProperty[]=[];
+  public actionGroups: bedrock.CfnAgent.AgentActionGroupProperty[] = [];
```

Comment:
(non-blocking) Consider keeping formatting changes in a separate PR

Reply:
sorry, was not my intention, was formatted automatically on save by my code formatter